src/components/EnrollmentsTable/columns.jsx

```diff
@@ -50,4 +50,53 @@ export const COLUMNS = [
     accessor: 'external_course_id',
     disableFilters: true,
   },
+  {
+    Header: 'Unenroll',
+    Cell: ({ row }) => {
+      const [isOpen, open, close] = useToggle(false);
+      const [show, setShow] = useState(false);
+      return (
+        <>
+          <Toast
+            onClose={() => setShow(false)}
+            show={show}
+          >
+            Unenroll Successfully !
```

Comment:
@anfbermudezme, I think it could be:
```suggestion
            Successfully unenrolled!
```